internal/provider/data_source_http.go

```diff
@@ -199,6 +212,25 @@ func (d *httpDataSource) Read(ctx context.Context, req datasource.ReadRequest, r
 		Transport: clonedTr,
 	}
 
+	// Override http client with retry logic if specified
+	if !model.RetryAttempts.IsNull() && !model.RetryAttempts.IsUnknown() {
+		tflog.Debug(ctx, "HEY COUCOU TOI")
+		tflog.Debug(ctx, fmt.Sprintf("RETRY: %d %d", model.RetryAttempts.ValueInt64(), model.RetryDelay.ValueInt64()))
```

Comment:
It might be worth adding text to make it clear what each of the values that are logged refer to.